src/types/config.ts

```diff
@@ -654,6 +655,7 @@ export type ExperimentalSettings = {
     ClientSideCertEnable: boolean;
     ClientSideCertCheck: string;
     EnableClickToReply: boolean;
+    CloudUserLimit: string;
```

Comment:
Is this supposed to be here twice? Looking at the server PR it looks like you have it declared twice in `GenerateClientConfig` as well.

Reply:
I can't say for sure, but I was following a pattern with a config that already existed. The ExperimentalSetting EnableClickToReply exists in 2 places, the client config flat map:
https://github.com/mattermost/mattermost-redux/blob/c60ae33b8d53b29c142d5cdfbc1ff13ac92d63da/src/types/config.ts#L106
And in the ExperimentalSettings object below it: 
https://github.com/mattermost/mattermost-redux/blob/c60ae33b8d53b29c142d5cdfbc1ff13ac92d63da/src/types/config.ts#L657

The client config `ExperimentalCloudUserLimit` config is the one I plan to use, I'm 0/5 on keeping the other one.

Reply:
It's possible it was pulled out of experimental at some point but no one removed it from the `ClientConfig` type. We're gonna need to make sure that process is followed by server changes going forward.

But yeah, just use the `ExperimentalCloudUserLimit` for now.

Reply:
Sounds good, removed the CloudUserLimit config in favour of ExperimentalCloudUserLimit